Makefile

```diff
@@ -3,8 +3,9 @@ MAKEFLAGS+=-r
 
 config=debug
 defines=standard
+cxxstd=default
```

Comment:
Just checked the AppVeyor files in the repository and it seems that AppVeyor is totally independent of the Makefile and my changes do not affect that at all?